scripts/generate_gauss_legendre_ref_data.py

```diff
@@ -21,27 +24,31 @@ def dump_reference_json():
     sys.path.insert(0, os.path.join(path, 'tests/'))
     data = []
     a = 0
-    for order in range(2,20):
-        for b in np.linspace(2,10,20):
-            x,w = get_leggauss(order, a, b)
-            data.append(dict(a=a,b=b,order=order,points=x.tolist(),weights=w.tolist()))
+    for order in range(2, 20):
+        for b in np.linspace(2, 10, 20):
+            x, w = get_leggauss(order, a, b)
+            data.append(dict(a=a, b=b, order=order,
+                             points=x.tolist(), weights=w.tolist()))
```

Comment:
is the list(np.array) or np.array().tolist() format preferred? @felixmusil

Reply:
I typically use .tolist() because it handles multidimensional arrays